P5/Source/Specs/unicodeProp.xml

```diff
@@ -0,0 +1,276 @@
+<?xml version="1.0" encoding="UTF-8"?><!-- Copyright TEI Consortium. Dual-licensed under CC-by and BSD2 licences See the file COPYING.txt for details $Date$ $Id$ -->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec ident="unicodeProp" module="gaiji" xml:id="UNICODEPROP" xmlns="http://www.tei-c.org/ns/1.0" xmlns:rng="http://relaxng.org/ns/structure/1.0"
+    xmlns:sch="http://purl.oclc.org/dsdl/schematron">
+    <gloss versionDate="2018-08-22" xml:lang="en">unicode property</gloss>
+    <desc versionDate="2018-08-22" xml:lang="en">holds the name and value of a normative or informative unicode character (or glyph) property as part of its attributes.</desc>
```

Comment:
```suggestion
    <desc versionDate="2020-01-28" xml:lang="en">provides a Unicode property for a character (or glyph).</desc>
```
The fact that these are provided as a name-value pair or via attributes does not really belong in the description.
Perhaps should be “provides a property defined by the Unicode standard for a character (or glyph).” or some such.

Reply:
the phrase `normative or…` is straight from the standard, i think there is something to be said for being in close allignment to avoid ambiguities. I m ok with removing the attributes reference.